src/components/DropdownV2/DropdownV2.js

```diff
@@ -169,5 +195,14 @@ export default class DropdownV2 extends React.Component {
         )}
       </Downshift>
     );
+    return title || helper ? (
+      <FormItem>
```

Comment:
While `<FormItem>` wrapping is something nice to consider in future, I'd be reluctant to change the behavior by `title`/`helper` existence. Wondering if we can use a fragment for now.